lib/src/page_list_viewport_gestures.dart

```diff
@@ -454,57 +670,85 @@ class DeprecatedPanAndScaleVelocityTracker {
   // Larger values result in the [FrictionSimulation] to accelerate faster and approach
   // zero slower, giving the impression of the simulation being "more slippery".
   // It was found through testing that other scroll systems seem to be use different dampening
   // factors for the vertical and horizontal components.
-  static const kVerticalDrag = 0.095;
-  static const kHorizontalDrag = 0.0425;
+  static const horizontalDragCoefficient = 250.0;
+  static const verticalDragCoefficient = 300.0;
+  static const staticFrictionCoefficient = 20.0;
+  // Mass is a redundant parameter, the ratio of m/c, mass to drag is important! Don't change:
+  static const mass = 100.0;
 
   PanningFrictionSimulation({
     required Offset position,
     required Offset velocity,
+    double initialVelocityIncreaseScalar = 1.0,
+    double dragIncreaseScalar = 1.0,
   })  : _position = position,
-        _velocity = velocity {
-    // Clamping bounds enforces a maximum instantaneous velocity of the viewport
-    // and in turn, how long the simulation will run. Larger numbers slide for longer,
-    // smaller numbers end more quickly.
-    _xSimulation = ClampedSimulation(
-      FrictionSimulation(
-        kHorizontalDrag,
-        _position.dx,
-        _velocity.dx,
-        // Horizontal friction simulation uses the default constantDeceleration.
-        //
-        // constantDeceleration impacts how smoothly the simulation comes to a stop
-        // near the end (when it smoothly slides to zero).
-      ),
-      dxMin: -3000,
-      dxMax: 3000,
-    );
-    _ySimulation = ClampedSimulation(
-      FrictionSimulation(
-        kVerticalDrag,
-        _position.dy,
-        _velocity.dy,
-        // Vertical friction is intentionally different from horizontal friction,
-        // based on empirical results.
-        //
-        // constantDeceleration impacts how smoothly the simulation comes to a stop
-        // near the end (when it smoothly slides to zero).
-        constantDeceleration: 2.35,
-      ),
-      dxMin: -2000, // Scroll down more slowly than scrolling up. A config found in some other scrolling systems.
-      dxMax: 3000,
-    );
+        _velocity = velocity,
+        _momentumSimInitialVelocityScalar = initialVelocityIncreaseScalar,
+        _dragIncreaseScalar = dragIncreaseScalar {
+    if (_velocity.dx.abs() > 0 && _velocity.dy.abs() > 0) {
+      // The simulation is not locked to an axis, it is in an arbitrary direction.
+      _xSimulation = FrictionAndFirstOrderDragMomentumSimulation(
+          staticFrictionCoefficient,
+          horizontalDragCoefficient * _dragIncreaseScalar,
+          mass,
+          _position.dx,
+          _velocity.distance,
+          math.cos(math.atan2(_velocity.dy, _velocity.dx)),
+          initialVelocityScalar: ViewportThresholdsAndScales.diagonalLaunchVelocityIncreaseScalar);
+      _ySimulation = FrictionAndFirstOrderDragMomentumSimulation(
```

Comment:
Please add a blank line above this. The original code should have had one, too, but it didn't.